cachelib/navy/common/tests/DeviceTest.cpp

```diff
@@ -196,6 +198,70 @@ TEST(Device, Stats) {
   device.getCounters({toCallback(visitor)});
 }
 
+// Test for FdpNvme Constructor
+TEST(FDP, InitializationTest) {
+  int nsId = 1;
+  uint32_t lbaShift = 12;
+  uint32_t maxTfrSize = 262144;
+  uint64_t startLba = 0;
+
+  NvmeData data(nsId, lbaShift, maxTfrSize, startLba);
+  struct nvme_fdp_ruh_status ruh_status;
+  ruh_status.nruhsd = 10;
+
+  EXPECT_NO_THROW(FdpNvme fdp = FdpNvme(data, &ruh_status));
+}
+
+// Test the Fdp Handle allocation
+TEST(FDP, FdpHandleAllocationTest) {
+  int nsId = 1;
+  uint32_t lbaShift = 12;
+  uint32_t maxTfrSize = 262144;
+  uint64_t startLba = 0;
+
+  NvmeData data(nsId, lbaShift, maxTfrSize, startLba);
+  struct nvme_fdp_ruh_status ruh_status;
+  ruh_status.nruhsd = 4;
```

Comment:
The ruh_status.ruhss is not initialized. This caused nullptr when running the test. can you please verify?